letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -162,7 +162,7 @@ SuSudo() {
 }
 
 # Sets the environment variable SUDO to be the name of the program or function
-# to call to get root access. If this script already has root privleges, SUDO
+# to call to get root access. If this script already has root privileges, SUDO
```

Comment:
This typo should not be fixed, because this file is frozen (for some reasons not worth getting into) and must not be changed.